Source/BusinessMomentsAdaptorImage/server.ts

```diff
@@ -23,6 +27,14 @@ export function createServer(rawDataStorage: IRawDataStorage) {
         try {
             await repo.Append(req.body);
             console.log(req.body);
+            const client = Client.forMicroservice('a87b3411-7f23-4ea3-9b74-f6fc7f7c5481')
```

Comment:
The client object will now be built every time the webhook request is called. I suggest you make this in index.ts and send it in as an parameter in the createServer function